goat_launch/launch/goat_odom.launch

```diff
@@ -0,0 +1,13 @@
+<launch>
+  <node pkg="robot_localization" type="ekf_localization_node" name="ekf_se" clear_params="true">
```

Comment:
rename this node to ekf_localization_node to follow convention in the rest of the repo